docs/concepts.md

```diff
@@ -51,8 +50,324 @@ The newly provisioned resource is automatically bound to the resource claim.
 To enable dynamic provisioning the administrator needs to create one or more resource class objects.
 
 ## Connection Secrets
-
```

Comment:
I believe we have blank lines following headers in other docs. Not sure it even renders differently on the site though.

Reply:
Doesn't render any differently, but does make it a little nicer to read the markdown. Needn't block getting this merged.